modules/base/mw.cx.SiteMapper.js

```
@@ -81,7 +82,12 @@ mw.cx.SiteMapper.prototype.getLanguageCodeForWikiDomain = function (domain, fall
  */
 mw.cx.SiteMapper.prototype.getApi = function (language, options) {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)